src/transform/tests/test_runner.rs

```diff
@@ -46,27 +46,41 @@ mod tests {
     const JSON: &str = "json";
     const TEST: &str = "test";
 
-    thread_local! {
-        static FULL_TRANSFORM_LIST: Vec<Box<dyn Transform>> = {
-            let features = OptimizerFeatures::default();
-            let typecheck_ctx = typecheck::empty_context();
-            let mut df_meta = DataflowMetainfo::default();
-            let mut transform_ctx = TransformCtx::local(&features, &typecheck_ctx, &mut df_meta, None);
+    const TEST_GLOBAL_ID: GlobalId = GlobalId::Transient(1234567);
```

Comment:
I don't think we'll ever get up to 12 million built-in things, but would it be possible to dynamically allocate this rather than hardcoding something? (Or adding an `assert!` early on in this testing function that this global ID isn't already in use?)

Reply:
I think hardcoding this in tests is beneficial, because then you can grep for this specific number when you see it in an error msg.

And I'm not afraid of id clashes, because having a million global objects would put a crazy load on envd, so this is not possible even in a test.